mailmerge/__main__.py

```diff
@@ -101,11 +106,9 @@ def main(sample, dry_run, limit, no_limit, resume,
         for _, row in enumerate_range(csv_database, start, stop):
             sender, recipients, message = template_message.render(row)
             sendmail_client.sendmail(sender, recipients, message)
-            print(">>> message {}".format(message_num))
-            print(message.as_string())
-            for filename in get_attachment_filenames(message):
-                print(">>> attached {}".format(filename))
-            print(">>> sent message {}".format(message_num))
+            print(TERM.reverse_bold_cyan(">>> message {}".format(message_num)))
+            print_message(message)
```

Comment:
Perhaps these two print statements should appear before actually sending the message?

Reply:
Good call!  25c42cc998e6cfb31f22071e9ee25d8579de41e8